graphs/possible_bipartition.py

```diff
@@ -1,12 +1,34 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
+from collections import defaultdict
 
-def possible_bipartition(dislikes):
+def possible_bipartition(dislikes: list[list[int]]):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
         Time Complexity: ?
         Space Complexity: ?
```

Comment:
⏱🪐 Time and space complexity?

Reply:
WHOOPS! excuse me while go and check my other projects LOL

Reply:
Thanks for updating 🤗

🟢